index.html

```diff
@@ -114,6 +114,10 @@ <h2 class="txt-center">
     $('.js-defaultServerIndexApp').on('click', function(){
       app.loadNewServer("http://localhost:18469/api/v1/");
     });
+    $('.js-translate').each(function() {
+      var translationKey = $(this).html();
```

Comment:
This won't work with phrases that have spaces in them, since the key can't have spaces.

I'm going to merge this, then update it so it also changes this text when the user changes their language in the settings view. I'll switch the phrase to a data attribute when I do that.